test

```diff
@@ -1,35 +1,28 @@
-#!/usr/bin/env bash
+#!/bin/bash -e
 
 set -eu
 
-source ./build
-
 SRC=$(find . -name '*.go' \
 	-not -path "./vendor/*")
 
-PKG=$(cd gopath/src/${REPO_PATH}; go list ./... | \
-	grep --invert-match vendor)
-
-# https://github.com/golang/go/issues/15067
-PKG_VET=$(cd gopath/src/${REPO_PATH}; go list ./... | \
-	grep --invert-match vendor | \
-	grep --invert-match internal/log)
+PKGS=$(go list ./... | grep -v /vendor)
+FORMATTABLE=$(ls -d */ | grep -v -E '(vendor/|examples/)')
```

Comment:
You can remove the exclusion of examples since there are is no examples folder in this project.